apps/web/src/components/PoolCandidatesTable/PoolCandidateTableFilterDialog.tsx

```diff
@@ -256,6 +257,30 @@ const PoolCandidateTableFilterDialog = ({
                   options={optionsData.poolCandidateStatus}
                 />
               </div>
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(3of5)">
+                <MultiSelectField
+                  id="skills"
+                  name="skills"
+                  label={formatMessage({
+                    defaultMessage: "Skill Filter",
+                    id: "GGaxMx",
+                  })}
+                  options={optionsData.skills}
+                  isLoading={rawGraphqlResults.skills.fetching}
+                />
+              </div>
+              <div data-h2-flex-item="base(1of1) p-tablet(1of2) laptop(1of3)">
+                <MultiSelectFieldBase
```

Comment:
The issue mentioned to maintain placement consistency between filter dialogs. I wasn't 100% on this so I elected to match its filter buddy, move it so it next to Skill Filter.

Reply:
I was iffy on it as the dialogs do look quite different 

User filter dialog 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7a0483aa-c578-45c3-8767-8f99ea0bd4d1)

Pool candidates filter dialog with the changes

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/6bd18f04-1744-49c4-8074-a939c0c8d8f5)